framework/src/main/java/com/retailmenot/scaffold/webelements/AbstractWebElement.java

```diff
@@ -298,6 +308,14 @@ protected WebElement getWebElement(boolean throwExceptionIfNotFound) {
             log.debug("Locating element on page: [%s]", by);
             return getWebDriverWrapper().findElement(by);
         } catch (NoSuchElementException e) {
+            // Pull the console error logs and report them. If none exist, create an empty list to bypass NPE
+            LogEntries logEntries = Optional.ofNullable(getWebDriverWrapper().manage().logs().get(LogType.BROWSER))
+                    .orElse(new LogEntries(Collections.emptyList()));
+
+            for (LogEntry entry : logEntries) {
+                log.error(new Date(entry.getTimestamp()) + " " + entry.getLevel() + " " + entry.getMessage());
```

Comment:
These are always being logged at an error level. Is there any reason you would want to match the entry.getLevel() to the level of the logging?

Reply:
Here, the `log.error` pertains to how it's surfaced through lombok, i.e. the `org.slf4j.Logger` interface (so things like warn, info, error), and will appear as such above the stacktrace on a failed test. The entry.getLevel() that's being reported pertains to the type of logging implemented with Selenium's `LogEntry` for pulling the logging from the console, i.e. the `java.util.logging.Level` class that has SEVERE, WARNING, etc.

What this means is that the error reported through the console could be any of the levels detailed from the `java.util.logging.Level` class. The purpose of reporting in this way is meant to be a QOL addition and not mapping synonymous levels between the two libraries; it is more along the lines of reporting that there is an error during the test failure and then displaying the level of the error from the console itself.